### ISSUE TITLE
Target_Temperatur_MK1 - gültiger Bereich < 18

### ISSUE THREAD

**Issue by andi1893** [BenPru/luxtronik#253]
Mein Mischerkreis 1 ist für die passive Kühlung aktiviert. 
An der Heizung habe ich 16 Grad als minimale Temperatur hinterlegt. 
Dieser Wert lässt sich allerdings in HA nicht einstellen, da er außerhalb des gültigen Bereichs liegt: 

Fehlermeldung: 
Der Wert 16 für number.luxtronik_320218_0267_cooling_target_temperature_mk1 liegt außerhalb des gültigen Bereichs 18.0 - 30.0.

**Comment by ChrisMisker** [comment]
Yes, I can set the cooling temperature to 15 with the app, but the home assistant integration is limiting it to 18 minimum. 
It should be dynamic, because there's a setting what the minimum value can be.

**Comment by BenPru** [comment]
@andi1893 @ChrisMisker What is the minimum and maximum in your app?
Can you laos check mk2?

**Comment by BenPru** [comment]
> Minimum is 10, max is 35. What do you mean with mk2 and mk3?

This config values exists for cooling temperatures:
number.luxtronik_*_cooling_target_temperature_mk1
number.luxtronik_*_cooling_target_temperature_mk2
number.luxtronik_*_cooling_target_temperature_mk3

**Comment by ChrisMisker** [comment]
> > Minimum is 10, max is 35. What do you mean with mk2 and mk3?
> 
> 
> 
> This config values exists for cooling temperatures:
> 
> number.luxtronik_*_cooling_target_temperature_mk1
> 
> number.luxtronik_*_cooling_target_temperature_mk2
> 
> number.luxtronik_*_cooling_target_temperature_mk3

I only have mk1.

**Comment by ChrisMisker** [comment]
I don't know how to suggest a change, but I'd say you would have to change this line:
Line 527 in number_entitities_predefined.py:
From
        native_min_value=18.0,
to
        native_min_value=15.0,

**Comment by rhammen** [comment]
@AJediIAm Looks like same/similar as #356

**Comment by AJediIAm** [comment]
I noticed this issue was reopened by mistake

**Comment by ChrisMisker** [comment]
I still think it would be better if the integration used the minimum cooling temperature configured in the heat pump itself instead of a hardcoded value.
On my Alpha Innotec (Luxtronik V3.92.3), the minimum cooling flow temperature is exposed as parameter P0993 (ID_Einst_min_VL_Kuehl). This parameter is only configurable through the installer menu on the heat pump, so changing it already requires installer-level access.
Since the controller itself already validates this setting, it seems more appropriate for the Home Assistant entity to use the current value of P0993 as the native_min_value for cooling_target_temperature_mk1 instead of always assuming 18 °C.
That would keep Home Assistant aligned with the actual controller configuration and avoid maintaining the same limit in two places. Users who leave the controller at 18 °C will still have a minimum of 18 °C, while users who intentionally change the installer setting (for example to 15 or 16 °C) would automatically get the correct range in Home Assistant as well.

**Comment by rhammen** [comment]
Yes, looks like @AJediIAm marked this one as duplicate of #356 and vice-versa, and then closed both ....
I will implement this.

**Comment by ChrisMisker** [comment]
@rhammen Just checking.. this isn't in the latest beta yet, is it? Would love to test this one! :)

**Comment by AJediIAm** [comment]
Sorry for closing too many issues. I think I lost track of some of the open ends.

- The minimal cooling temperature is a safety parameter and should NOT be used for the thermostat. It can physically damage the system.
- The target cooling temperature is a good candidate for the thermostat since it controls the "strength" of the cooling. 
- The room thermostat shows the same target temperature for heating and cooling. This value is used to activate cooling, so it is very relevant.

**Comment by rhammen** [comment]
> Sorry for closing too many issues. I think I lost track of some of the open ends.
> 
>     * The minimal cooling temperature is a safety parameter and should NOT be used for the thermostat. It can physically damage the system.
> 
>     * The target cooling temperature is a good candidate for the thermostat since it controls the "strength" of the cooling.
> 
>     * The room thermostat shows the same target temperature for heating and cooling. This value is used to activate cooling, so it is very relevant.

@AJediIAm 
I'm confused by your remarks about the thermostat:

The request of @ChrisMisker is: "I still think it would be better if the integration used the minimum cooling temperature (minimum value vor entities COOLING_TARGET_TEMPERATURE_MKx, with x=1,2,3) configured in the heat pump itself instead of a hardcoded value.".
So it is about the lowest value for entities COOLING_TARGET_TEMPERATURE_MKx that you are allowed to set in the integration.
He proposes that instead of fixed value of 18.0 C, it should use the value of P0993_COOLING_MIN_FLOW_OUT_TEMPERATURE.

That request has nothing to do with the cooling thermostat of this integration.
The thermostat of this integration does not use COOLING_TARGET_TEMPERATURE_MKx. (It is the luxtronik controller that uses it.)
The cooling thermostat of this integration uses (dependant of the RBE type) either the room temperature, or the outside temperature threshold for cooling.
I am not changing the cooling thermostat with this request.

Note: the user CAN change the value of P0993_COOLING_MIN_FLOW_OUT_TEMPERATURE using the integration (#633) , but that is the users responsibility. Using this integration, a user cannot set its value lower than 17.0 C. 
If the installer has set a lower value, that will be used.
Users where the installer set a lower temperature need to be careful not to change its value using the integration, because once they do, they cannot set it lower than 17.0 C using the integration. --> that is maybe something for the documentation.

**Comment by rhammen** [comment]
> [@rhammen](https://github.com/rhammen) Just checking.. this isn't in the latest beta yet, is it? Would love to test this one! :)

@ChrisMisker Try pre-release [2026.06.29](https://github.com/BenPru/luxtronik/releases/tag/2026.06.29)

**Comment by ChrisMisker** [comment]
Small error, I think... the number set in the heat pump is 15, but it becomes 150.

<img width="661" height="90" alt="Image" src="https://github.com/user-attachments/assets/123f3b2e-a363-4e89-ba21-33a2a9c9a2c1" />

**Comment by rhammen** [comment]
> Small error, I think... the number set in the heat pump is 15, but it becomes 150.
> <img alt="Image" width="661" height="90" src="https://private-user-images.githubusercontent.com/37826263/614779838-123f3b2e-a363-4e89-ba21-33a2a9c9a2c1.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODI3NjE2NTUsIm5iZiI6MTc4Mjc2MTM1NSwicGF0aCI6Ii8zNzgyNjI2My82MTQ3Nzk4MzgtMTIzZjNiMmUtYTM2My00ZTg5LWJhMjEtMzNhMmE5YzlhMmMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNjI5VDE5MjkxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExNzdkMGY1ZGM0OTE1ZjUxMTJjMGQxZmQ4MjJhMjRhMzM1NmYzNDU4YTNlMWFkMGQ4NjlhN2U2Zjc0NjRmOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.I_R4lbox1knmm3g65Aa7PoJ4YIkBMEq4YsA1ZMvPIk4">

OK, thanks, I see.
I will fix that tomorrow evening.

**Comment by AJediIAm** [comment]
> > Sorry for closing too many issues. I think I lost track of some of the open ends.
> > ```
> > * The minimal cooling temperature is a safety parameter and should NOT be used for the thermostat. It can physically damage the system.
> > 
> > * The target cooling temperature is a good candidate for the thermostat since it controls the "strength" of the cooling.
> > 
> > * The room thermostat shows the same target temperature for heating and cooling. This value is used to activate cooling, so it is very relevant.
> > ```
> 
> [@AJediIAm](https://github.com/AJediIAm) I'm confused by your remarks about the thermostat:
> 
> The request of [@ChrisMisker](https://github.com/ChrisMisker) is: "I still think it would be better if the integration used the minimum cooling temperature (minimum value vor entities COOLING_TARGET_TEMPERATURE_MKx, with x=1,2,3) configured in the heat pump itself instead of a hardcoded value.". So it is about the lowest value for entities COOLING_TARGET_TEMPERATURE_MKx that you are allowed to set in the integration. He proposes that instead of fixed value of 18.0 C, it should use the value of P0993_COOLING_MIN_FLOW_OUT_TEMPERATURE.
> 
I mis understood.
What you describe is correct .

**Comment by rhammen** [comment]
> Small error, I think... the number set in the heat pump is 15, but it becomes 150.
> <img alt="Image" width="661" height="90" src="https://private-user-images.githubusercontent.com/37826263/614779838-123f3b2e-a363-4e89-ba21-33a2a9c9a2c1.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODI4NTQ0OTEsIm5iZiI6MTc4Mjg1NDE5MSwicGF0aCI6Ii8zNzgyNjI2My82MTQ3Nzk4MzgtMTIzZjNiMmUtYTM2My00ZTg5LWJhMjEtMzNhMmE5YzlhMmMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNjMwVDIxMTYzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2MzQzODgzMmZiNjlmOTQyNTc0ODc5ZjIyMDIzMDU3Y2ZhNjVhMzUzNDAxZTA0N2EwNDE0ZGFhYzQzNmZkNDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.LW3BMlRziPTFo3tWFXSZz2rtjl_9eu6s5RYHAkvTVnw">

Fixed in [2026.06.30](https://github.com/BenPru/luxtronik/releases/tag/2026.06.30)

**Comment by rhammen** [comment]
Fixed with #660
